src/logic/safe/store/reducer/__tests__/gatewayTransactions.test.ts

```diff
@@ -214,6 +126,13 @@ describe('gatewayTransactionsReducer', () => {
         },
         ...MOCK_TX_META,
       }
+      const mockQueuedTx2: Transaction = {
+        ...mockQueuedTx,
```

Comment:
It doesn't need an `id` here. We could theoretically remove all of them from the tests now (I think).